cypress/e2e/create_tx.cy.js

```diff
@@ -103,10 +144,11 @@ describe('Queue a transaction on 1/N', () => {
       cy.get('input[type="checkbox"]').should('not.exist')
     })
 
-    cy.contains('Submit').click()
+    // We don't want to keep queing tx for now
+    //cy.contains('Submit').click()
```

Comment:
Maybe I'm shortsighted here but what is the problem of keep queuing?

Reply:
This will otherwise drain our 1Balance.

Reply:
I don't know if a safe has a ton of queued tx is stable (long to load or something like that). As long we keep this test out of the CI I can enable it again

Reply:
@iamacook The test ends up just queueing a transaction, not relaying it.

Reply:
> @iamacook The test ends up just queueing a transaction, not relaying it.

Disregard my comment 🙈